Packs/ThreatGrid/TestPlaybooks/playbook-Threat_Grid_Test.yml

```diff
@@ -1,387 +1,2986 @@
 id: ThreatGridTest
 version: -1
-name: ThreatGridTest
+name: playbook-ThreatGridTry_Test
```

Comment:
why change it?
If this is a different test playbook from the one used for v1, please add it as a separate file, and restore the previous.